app/views/claims/information_provided.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">How we will use the information you provide</h1>
+    <p class="govuk-body">
+      To process your claim we will check the information you provide against the information the
+      Department for Education hold.
+    </p>
+
+    <p class="govuk-body">To determine if your claim is valid we will:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        check your eligibility against the Database of Qualified Teachers. You may have used with
+        this in the past as the Teacher Self-Service Portal
+      </li>
+      <li>check your eligibility against the Teachers Pension Service</li>
+      <li>check your eligibility with any previous or current schools you have worked for</li>
+    </ul>
+
+    <p class="govuk-body">To progress and pay your claim we will:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>pay you as the Department for Education. You will see this on a future P60</li>
+      <li>send some of your personal information to HMRC</li>
+      <li>pay a tax, student loan and national insurance contribution to HMRC</li>
+    </ul>
+
+    <p class="govuk-body">
+      We will send you details about these payments once your application is complete.
+    </p>
+
+    <%= link_to "Continue to GOV.UK Verify", new_verify_authentications_path, class: "govuk-button" %>
```

Comment:
It's not clear what this next step actually is now. It sounds like we're done at this point from the text above.

Reply:
On the mock up this button reads "Continue". The page the user links to explains what verify is and how it works. Probably best if we drop the `to GOV.UK Verify` for now.

Reply:
Yeah I thought it was an omission on the mockup. I'll remove it

Reply:
My comment was less about the text on the Continue button, but more about the expectation of what happens next. This page talks about making the claim as if we're now done, so it's going to be jarring for a user to still have 10ish steps to go, including a long interaction with Verify. I think there needs to be some words about how next we'll ask the user some information to verify their identity and collect details about how to contact and pay them.